package.json

```diff
@@ -51,6 +51,7 @@
   },
   "dependencies": {
     "debug": "^4.3.1",
+    "node-saml": "file:../node-saml",
```

Comment:
this seems to expect a certain folder structure which is fine for local development during the transition but should probably at least be a import from a git hash before the PR lands (and quickly be changed to the actual npm package)

Reply:
Absolutely. I wasn't going to land it like this. I wanted to get an NPM package in here; thus the other PR over there. I put this up so you could all see why I made the changes I did over on the node-saml project. If everything else looks good here, I'll adjust this reference as soon as node-saml is released as a package.